Tests/APNSKitTests/APNSKitTests.swift

```diff
@@ -3,10 +3,6 @@ import XCTest
 
 final class APNSKitTests: XCTestCase {
     func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
-        XCTAssertEqual(apns_kit().text, "Hello, World!")
     }
 
     static var allTests = [
```

Comment:
We should rm this and use `swift test --generate-linuxmain` instead.

Reply:
@tanner0101 the whole file?

Reply:
Just the `allTests` array. The command `swift test --generate-linuxmain` will use Swift / Objc magic to generate a `XCTestManifests.swift` for each test module and a `LinuxMain.swift` that combines them. 
Just make sure to add the newly generated files to version control after running the command.

Reply:
OKay I think I did it right. 🤞